qiita_pet/templates/study_description.html

```diff
@@ -548,6 +568,7 @@
         <a class="btn btn-default glyphicon glyphicon-edit" href="/study/edit/{{study.id}}" title="Edit the study information" style="margin:5px; word-spacing: -10px;"> Edit</a>
       {% end %}
       <a href="/study/upload/{{study.id}}" class="btn btn-default glyphicon glyphicon-upload" title="Upload study files" style="margin:5px; word-spacing: -10px;"> Upload</a>
+      <a class="btn btn-danger glyphicon glyphicon-trash" data-toggle="modal" data-target="#delete-study"></a>
```

Comment:
Since there are two "trash" icons (when viewing the sample template), I think this button should have a "Delete Study" label associated with it.